app/src/main/java/chat/rocket/android/push/interactors/DefaultPushInteractor.java

```diff
@@ -1,19 +0,0 @@
-package chat.rocket.android.push.interactors;
-
-import chat.rocket.android.helper.ServerPolicyHelper;
-import chat.rocket.android.model.ServerConfig;
-import chat.rocket.android.realm_helper.RealmStore;
-
-public class DefaultPushInteractor implements PushInteractor {
```

Comment:
The idea is to make `PushNotificationHandler` unaware of `Realm` stores and etc. On the long run, we want `Realm` to be an implementation detail and not something that's everywhere in the app.

So `PushNotificationHandler` only knows about a `PushInteractor` interface and the `DefaultPushInteractor` implements that using `Realm`.